internal/controller/suite_test.go

```diff
@@ -38,16 +42,19 @@ var (
 	cfg       *rest.Config
 	k8sClient client.Client
 	testEnv   *envtest.Environment
+	ctx       context.Context
+	cancel    context.CancelFunc
 )
 
-func TestControllers(t *testing.T) {
+func TestControllers(t *testing.T) { //nolint:paralleltest
 	RegisterFailHandler(Fail)
 
 	RunSpecs(t, "Controller Suite")
 }
 
 var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
+	ctx, cancel = context.WithCancel(context.TODO())
```

Comment:
Prefer Background over TODO

Reply:
I had the opposite in my head, but for tests it is indeed true.